pulp_puppet_plugins/pulp_puppet/plugins/db/models.py

```diff
@@ -68,7 +68,7 @@ class Module(FileContentUnit):
     tag_list = ListField()
 
     # Generated at the file level
-    checksum = StringField(required=True)
```

Comment:
Is the `required=True` dropped because lazy may not have it be available until later?